test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -453,5 +458,18 @@ private static void VerifyLogsHasLineInfo(string[] logs)
                 }
             }
         }
+
+        private static void VerifyNoChangesOnInputFiles(FileInfo[] inputFiles, string[] skippableInputs)
+        {
+            foreach(var file in inputFiles)
+            {
+                if (skippableInputs.Contains(file.Name))
+                {
+                    continue;
+                }
+                Assert.True(File.Exists(file.FullName), $"Input file {file.Name} has been deleted");
+                Assert.True(file.LastWriteTime == new FileInfo(file.FullName).LastWriteTime, $"Input file {file.Name} has been changed");
```

Comment:
Seems like the `LastWriteTime` assertion is always true?

Reply:
No, the `FileInfo` will not change once it has been created

---
In reply to: [284090364](https://github.com/dotnet/docfx/pull/4518#discussion_r284090364) [](ancestors = 284090364)

Reply:
@928PJY , better not relying on this implementation details, suggest using a `Dictionary<string, DateTime>`

Reply:
sure

---
In reply to: [284527224](https://github.com/dotnet/docfx/pull/4518#discussion_r284527224) [](ancestors = 284527224)